package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@launchdarkly/ldcli",
```

Comment:
I didn't include a version assuming release-please is able to set it. During testing with the manual-release action, I had to keep it in sync with the go version release and had to bump it for each release to npm. I don't know how release-please sets the version.